packages/blocks/src/surface-block/managers/edit-session.ts

```diff
@@ -280,9 +259,28 @@ export class EditSessionStorage {
   }
 }
 
-export function isLastPropType(
-  type: EdgelessElementType
-): type is keyof LastProps {
+function extractProps(
```

Comment:
Please double check unnecessary exports. And utils not using `this` should better be defined as plain function rather than class methods.

Reply:
OK, Thanks for mentioning!